test/ecto/dev_logger/printable_parameter_test.exs

```diff
@@ -88,6 +88,9 @@ defmodule Ecto.DevLogger.PrintableParameterTest do
              %Postgrex.INET{address: {127, 0, 0, 1}, netmask: nil}
            ]) == "'{127.0.0.1/24,127.0.0.1}'"
 
+    assert to_expression([%Geo.Point{coordinates: {44.21587, -87.5947}, srid: 4326, properties: %{}}]) ==
+      "'{\"%Geo.Point{coordinates: {44.21587, -87.5947}, properties: %{}, srid: 4326}\"}'"
```

Comment:
This is not a valid string representation of geo point.
I didn't use postgis, so I can't help here, but quick googling shows, that it should be represented like `SRID=4326;POINT(44.21587 -87.5947)`